test/parallel/test-process-env-deprecation.js

```diff
@@ -12,5 +12,9 @@ common.expectWarning(
   'DEP0104'
 );
 
+// Make sure setting a valid environment variable doesn't
+// result in warning being suppressed, see:
```

Comment:
@joyeecheung confirmed that this failure wasn't coverage related, setting an environment variable correctly would suppress future warnings; easy fix is simply calling `env->EmitProcessEnvWarning()` last.

Reply:
@refack not quite, there are a couple `addon` tests that fail; I think related to the inspector session creating resources that break a few assertions.

I would be happy to help isolate these, and maybe we could use `TEST_TO_SKIP_FOR_COVERAGE` to remove them, then maybe we could get to the point where @mhdawson and I can abort on failure for coverage tests?

Would folks be okay with keeping this patch in, if @mhdawson and I work on making this pull request run coverage without ignoring errors?